packages/elements/package.json

```diff
@@ -54,7 +54,7 @@
     "react-google-map": "^3.1.1",
     "react-google-maps-loader": "^4.2.5",
     "react-icons": "^3.7.0",
-    "react-table": "7.0.0-alpha.35",
+    "react-table": "7.0.4",
```

Comment:
Upgrade to stable version of react-table to add Footer to the table
https://react-table.js.org/examples/simple#footers

Version 7.0.0-alpha.35 didn't had this feature.